oracle/README.md

```diff
@@ -321,6 +322,43 @@ Verifies the database is available and accepting connections.
 
 ## Troubleshooting
 
+### Common Problems
+#### Oracle Instant Client
+- Verify that both the Oracle Instant Client and SDK files are located in the same directory.
```

Comment:
Let's add an exemple tree structure.

Reply:
I added an example tree structure and removed the sdk sub files since it made the example unnecessarily large